src/lib/copy.ts

```diff
@@ -1,3 +1,8 @@
+interface Options {
+    text: string;
+    events: string | string[];
```

Comment:
Should make this optional
```suggestion
    events?: string | string[];
```